.changeset/old-jeans-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+"@digdir/designsystemet-react": patch
+---
+
+Export all utilities
```

Comment:
```suggestion
feat: Export all utilities
```

Reply:
We have not used this notation in any other changesets, any reason you want it here?
We decided earlier that selecting the correct form of "major/minor/patch" is good enough, as the list it spits out is already sorted.

Reply:
ah, I forgot :)